staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/admission_test.go

```diff
@@ -604,7 +604,7 @@ func webhookHandler(w http.ResponseWriter, r *http.Request) {
 	case "/allow":
```

Comment:
add a "/nilResponse" section (which returns an object with a nil response field) & a test that exercises it, or a TODO?

Reply:
Done